test/utils.ts

```diff
@@ -77,10 +77,14 @@
 
 /** Builds the sample binary module code. We need to do this because the source maps have absolute paths so they are not portable between machines, and while we could do deterministic with source maps, that's way more complicated and everywhere we build has dotnet already anyways */
 export function BuildBinaryModuleMock(): void {
-    console.log("==BUILDING: Binary Module Mock==");
     const projectPath = path.resolve(`${__dirname}/../../test/mocks/BinaryModule/BinaryModule.csproj`); //Relative to "out/test" when testing.
-    const buildResult = execSync(`dotnet publish ${projectPath}`);
-    console.log(buildResult.toString());
+    try {
+        execSync(`dotnet publish ${projectPath}`, {
```

Comment:
I updated it to remove the log which is why it complained again, I could do it as a spawn but it's just not worth the effort. It shouldn't complain after this since I flagged it as "used for tests"